### PR TITLE
[Improve] [doc] Display plugins in the form of a table

### PR DESCRIPTION
**Pull request by zhaomin1423**
## Purpose of this pull request

Display plugins in the form of a table

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in you PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by zhaomin1423** [comment]
HI, Could you help me to review it. @asdf2014 @CalvinKirs

**Review by CalvinKirs — COMMENTED** [review]
It would be better if the table spacing could be aligned, also, there seems to be a problem with the link citation

**Comment by CalvinKirs** [comment]
like this 
image

File: README.md [hunk 1]
```diff
@@ -62,19 +62,67 @@ processing plug-in, because the whole system is easy to expand.
...
+| |MongoDB|[doc](./docs/env/spark/configuration/source-plugins/MongoDB.md)|[doc](./docs/env/spark/configuration/sink-plugins/MongoDB.md)|
+| |neo4j|[doc](./docs/env/spark/configuration/source-plugins/neo4j.md)|[doc](./docs/env/spark/configuration/sink-plugins/neo4j.md)|
+| |Phoenix|[doc](./docs/env/spark/configuration/source-plugins/Phoenix.md)|[doc](./docs/env/spark/configuration/sink-plugins/Phoenix.md)|
+| |Redis|[doc](./docs/env/spark/configuration/source-plugins/Redis.md)|[doc](./docs/env/spark/configuration/sink-plugins/Redis.md)|
+| |Tidb|[doc](./docs/env/spark/configuration/source-plugins/Tidb.md)|[doc](./docs/env/spark/configuration/sink-plugins/Tidb.md)|
+| |Clickhouse| |[doc](./docs/env/spark/configuration/sink-plugins/Clickhouse.md)|
+| |Doris| |[doc](./docs/env/spark/configuration/sink-plugins/Doris.md)|
+| |Email| |[doc](./docs/env/spark/configuration/sink-plugins/Email.md)|
+| |Hbase| |[doc](./docs/env/spark/configuration/sink-plugins/Hbase.md)|
+| |Kafka| |[doc](./docs/env/spark/configuration/sink-plugins/Kafka.md)|
```
Comment (wuchunfu) [inline README.md]: This should be `./docs/en/` not `./docs/env/`

Comment (wuchunfu) [reply README.md]: @zhaomin1423 Please check it as a whole and fix it, thanks

**Comment by zhaomin1423** [comment]
> It would be better if the table spacing could be aligned, also, there seems to be a problem with the link citation

Thanks, I updated it. But, There is some misalignment in the display. In fact, it is aligned

File: README.md (hunk 1 above)
Comment (zhaomin1423) [reply README.md]: Thanks. I updated.

**Comment by CalvinKirs** [comment]
LGTM 
image
It would be more beautiful if these tables could also be aligned.

**Comment by zhaomin1423** [comment]
> 

The style can't work in github, but it can work well in idea. How can I do?

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Comment by CalvinKirs** [comment]
> > 
> 
> The style can't work in github, but it can work well in idea. How can I do?

Thanks for your contribution, but these tags are generally not recommended in MD. I'll fix it later, thanks again for your contribution.

**Comment by zhaomin1423** [comment]
> > > 
> > 
> > 
> > The style can't work in github, but it can work well in idea. How can I do?
> 
> Thanks for your contribution, but these tags are generally not recommended in MD. I'll fix it later, thanks again for your contribution.

Thanks for your review.

**Comment by asdf2014** [comment]
@zhaomin1423 Thanks for your contribution, let’s link this to issue #1317, and I have raised up another PR #1348 to update the Chinese README doc

**Comment by zhaomin1423** [comment]
> @zhaomin1423 Thanks for your contribution, let’s link this to issue #1317, and I have raised up another PR #1348 to update the Chinese README doc

Thanks，we may need a issue for add the Chinese doc of plugins.

**Pull request merged**